examples/image-slider.amp.html

```diff
@@ -78,6 +78,54 @@
       transform: translate(50%, -50%);
     }
 
+    .reversed-hint .amp-image-slider-hint-left {
+      background-size: 64px 32px;
+      width: 64px;
+      height: 32px;
+      margin-right: 30px;
+      background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 8'%3E%3Cg data-name='Layer 2'%3E%3Cpath d='M12 5H0V3h12V0l4 4-4 4z' fill='%23fff' data-name='Layer 1'/%3E%3C/g%3E%3C/svg%3E");
```

Comment:
The `data-name`s seem to be extra.

Reply:
Leftovers from Sketch (as the original SVG is provided by Abby). Could remove that